org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/capabilities/InitialServerCapabilities.java

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright (c) 2018 Red Hat, Inc. and others.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *  Red Hat Inc. - initial API and implementation
+ */
+
+package org.eclipse.lsp4xml.settings.capabilities;
+
+import java.util.Arrays;
+
+import org.eclipse.lsp4j.CompletionOptions;
+import org.eclipse.lsp4j.DocumentLinkOptions;
+import org.eclipse.lsp4j.ServerCapabilities;
+import org.eclipse.lsp4j.TextDocumentSyncKind;
+
+/**
+ * All default capabilities of this server
+ */
+public class InitialServerCapabilities {
+
+	public static CompletionOptions DEFAULT_COMPLETION_OPTIONS = new CompletionOptions(false, Arrays.asList(".", ":", "<", "\"", "=", "/"));
+	public static TextDocumentSyncKind DEFAULT_SYNC_OPTION = TextDocumentSyncKind.Full;
+	public static DocumentLinkOptions DEFAULT_LINK_OPTIONS = new DocumentLinkOptions(true);
+
+	/**
+	 * Returns all default server capabilities, unless a dynamic option is selected
+	 * @param manager
+	 * @return
+	 */
+	public static ServerCapabilities getNonDynamicServerCapabilities(XMLCapabilityManager manager) {
```

Comment:
getNonDynamicServerCapabilities(ClientCapabilitiesWrapper clientCapabilities)

We just don't need a XMLCapabilityManager here